_includes/pbc/all/install-features/202407.0/install-glue-api/install-the-product-glue-api.md

```diff
@@ -0,0 +1,601 @@
+| ProductTaxSetsRestApi | vendor/spryker/product-tax-sets-rest-api |
+| ProductsCategoriesResourceRelationship | vendor/spryker/products-categories-resource-relationship |
+| ProductAttributesRestApi |vendor/spryker/product-attributes-rest-api|
+
+{% endinfo_block %}
+
+## 2) Set up configuration
+
+{% info_block infoBox %}
+
+You can control whether the `abstract-products` get the `concrete-products` as a relationship by default with the `ProductsRestApiConfig::ALLOW_PRODUCT_CONCRETE_EAGER_RELATIONSHIP` config setting.
+
+{% endinfo_block %}
+
+**src/Pyz/Glue/ProductsRestApi/ProductsRestApiConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Glue\ProductsRestApi;
+
+use Spryker\Glue\ProductsRestApi\ProductsRestApiConfig as SprykerProductsRestApiConfig;
+
+class ProductsRestApiConfig extends SprykerProductsRestApiConfig
+{
+    public const ALLOW_PRODUCT_CONCRETE_EAGER_RELATIONSHIP = false;
+}
+```
+
+{% info_block errorBox %}
+
+We recommend setting `ALLOW_PRODUCT_CONCRETE_EAGER_RELATIONSHIP` to `false`.  
+
+Using `ALLOW_PRODUCT_CONCRETE_EAGER_RELATIONSHIP = true` in combination with `ConcreteProductsByProductConcreteIdsResourceRelationshipPlugin`, which is described later in this document, results in duplicated relationships.
+
+We also recommend using `ConcreteProductsByProductConcreteIdsResourceRelationshipPlugin` with `ALLOW_PRODUCT_CONCRETE_EAGER_RELATIONSHIP = false`. The config setting exists for BC reasons only.
+
+{% endinfo_block %}
+
+
+## 3) Set up database schema and transfer objects
+
+Run the following commands to update the database and generate entity and transfer changes:
+
+```bash
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following changes have occurred in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+| --- | --- | --- | --- |
+| AbstractProductsRestAttributesTransfer | class | created | src/Generated/Shared/Transfer/AbstractProductsRestAttributesTransfer |
+| ConcreteProductsRestAttributesTransfer | class | created | src/Generated/Shared/Transfer/ConcreteProductsRestAttributesTransfer|
+| RestProductImageSetsAttributesTransfer|  class|  created | src/Generated/Shared/Transfer/RestProductImageSetsAttributesTransfer|
+| RestProductImageSetTransfer|  class|  created  | src/Generated/Shared/Transfer/RestProductImageSetTransfer|
+| RestImagesAttributesTransfer |  class |  created | src/Generated/Shared/Transfer/RestImagesAttributesTransfer|
+| RestProductPriceAttributesTransfer|  class|  created | src/Generated/Shared/Transfer/RestProductPriceAttributesTransfer|
+| RestProductPricesAttributesTransfer | class|  created | src/Generated/Shared/Transfer/RestProductPricesAttributesTransfer|
+| RestCurrencyTransfer|  class | created | src/Generated/Shared/Transfer/RestCurrencyTransfer|
+| RestProductManagementAttributeAttributes|  class|  created|  src/Generated/Shared/Transfer/RestProductManagementAttributeAttributesTransfer|
```

Comment:
We need to remove the suffix `Transfer` from the transfers above or add it to the transfers below.